test/built-ins/TypedArray/prototype/values/make-out-of-bounds-after-exhausted.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2024 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.values
+description: >
+  Calling next on an out-of-bounds typedarray throws no error when iterator exhausted.
+features: [TypedArray, resizable-arraybuffer]
```

Comment:
Same as previous comment, just a suggestion.

```suggestion
info: |
  %TypedArray%.prototype.values ( )

  ...
  3. Return CreateArrayIterator(O, VALUE).
  
features: [TypedArray, resizable-arraybuffer]
```

Reply:
Likewise, assuming this should stay as is for the time being.